### ISSUE TITLE
Hangouts: Add Monero.town

### ISSUE THREAD

**Issue by HardenedSteel** [monero-project/monero-site#2350]
add monero.town lemmy instance to hangouts page

**Comment by nahuhh** [comment]
Tor working for me now

**Comment by HardenedSteel** [comment]
isn't getmonero.org already behind the cloudflare?

**Comment by nahuhh** [comment]
Yes

**Comment by monerobull** [comment]
> which means federation works on a whitelist mode

This is false. I don't know why the instance you are on hasn't federated in 3 months but other instances have no issues.

**Comment by monerobull** [comment]
> > > which means federation works on a whitelist mode
> > 
> > 
> > This is false. I don't know why the instance you are on hasn't federated in 3 months but other instances have no issues.
> 
> Care to explain this [message](https://matrix.to/#/!WzzKmkfUkXPHFERgvm:matrix.org/$SiPV8XoWCGcWBxRoFcHhU93BF-I9hNZcI6yoUHeGtL4?via=matrix.org&via=monero.social&via=libera.chat), then?
> 
> IRC: https://libera.monerologs.net/monero-community/20240715#c398496
> 
> While we're at it, why do lemmy.cafe's posts don't show up on .town/c/monero, but lemmy.zip's do? Hint: .zip is behind CF, cafe isn't.

I was wrong about the federation not working, it was the cloudflare captcha that breaks it, not cloudflare itself. 



> Yet more messages written by you, alluding to such whitelist, this one is more recent than the one above. False much, eh? Quoting Monero's GitLab [comment](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/481#note_25594):
> 
> > Btw do you want to mod on monero.town? There is very little to do because of the whitelist but

This is completely unrelated and talking about the sign-up whitelist that prevents bot accounts and helps keeping the instance alive.
